main/src/ca/uwaterloo/flix/language/ast/shared/TraitUsageKind.scala

```diff
@@ -23,6 +23,12 @@ sealed trait TraitUsageKind
   * Used to indicate the context of an undefined trait
   */
 object TraitUsageKind {
+  /**
+    * Represents a trait use in a call
+    * e.g. let res = E...
+    */
+  case object Call extends TraitUsageKind
```

Comment:
Call -> Expr

(You cannot be sure this is a use in a call, it could just be a reference).